src/tidy.zig

```diff
@@ -89,8 +110,8 @@ fn is_naughty(path: []const u8) bool {
 
 fn find_long_line(file_text: []const u8) !?usize {
     var line_iterator = mem.split(u8, file_text, "\n");
-    var line_number: usize = 0;
-    while (line_iterator.next()) |line| : (line_number += 1) {
+    var line_index: usize = 0;
```

Comment:
Finally figured out a resonable naming scheme for user-visible line numbers: `line_index` is zero-based, `linu_number` is one-based.